utils/shellcommand/hostsfile.go

```diff
@@ -14,18 +14,16 @@
 
 package shellcommand
 
-import "fmt"
-
-const (
```

Comment:
i thought we should set different  alias, otherwise it will be deleted by mistake, its a bug that i meet before.